environments/aat.yml

```diff
@@ -3,7 +3,11 @@ ns:
 mx:
 
 A:
-  - name: "cvp-recording"  
+  - name: "cvp-recordings"  
```

Comment:
isn't this the same as the record below it?

Reply:
with added "S"

Reply:
testing the AAT apply and will remove if it kicks in later